### PR TITLE
[PATCH v3]  example: ipsec_crypto: enhance runtime scripts 

### PR DESCRIPTION
**Pull request by asasidharan**
This PR enhances the ipsec_crypto application runtime scripts similar to that being done for ipsec_api application in https://github.com/OpenDataPlane/odp/pull/1265.

For the ipsec_crypto application enabling all the run scripts by default. In the pktio_env script based on the IPSEC_APP_MODE, skipping the live / router mode tests in the CI.

### CODE REVIEW

**Review by MatiasElo — COMMENTED** [review]
Couple minor comments but otherwise looks good.
- `pktio_env` is missing from .gitignore
- Rebase required

File: example/ipsec_crypto/odp_ipsec_crypto_run_ah_in.sh [hunk 1]
```diff
@@ -4,9 +4,40 @@
...
 #  - 10 packets
 #  - Specify API mode on command line
-./odp_ipsec_crypto -i loop1,loop2 \
--r 192.168.111.2/32,loop1,08:00:27:76:B5:E0 \
--p 192.168.222.0/24,192.168.111.0/24,in,ah \
--a 192.168.222.2,192.168.111.2,md5,300,27f6d123d7077b361662fc6e451f65d8 \
--s 192.168.222.2,192.168.111.2,loop2,loop1,10,100 \
--c 2 -m $1
+
+# IPSEC_APP_MODE: 0 - STAND ALONE, 1 - LIVE, 2 - ROUTER
```
[the remark this answers is not in the document]
Comment (asasidharan) [reply example/ipsec_crypto/odp_ipsec_crypto_run_ah_in.sh:8]: Ack. Will fix this in v2.

File: example/ipsec_crypto/odp_ipsec_crypto_run_router.sh [hunk 2]
```diff
@@ -3,7 +3,55 @@
...
+# this just turns off output buffering so that you still get periodic
+# output while piping to tee, as long as stdbuf is available.
+STDBUF="`which stdbuf 2>/dev/null` -o 0" || STDBUF=
+LOG=odp_ipsec_crypto_tmp.log
+PID=app_pid
+
+($STDBUF \
+ ./odp_ipsec_crypto -i $IF0,$IF2 \
+	-r 192.168.111.2/32,$IF0,$MAC_ADDR_IF1 \
+	-r 192.168.222.2/32,$IF2,$MAC_ADDR_IF3 \
```
[the remark this answers is not in the document]
Comment (asasidharan) [reply example/ipsec_crypto/odp_ipsec_crypto_run_router.sh:29]: Ack. Will fix this in v2.

**Comment by asasidharan** [comment]
With v2:
- Rebased PR to latest master branch.
- Fixed typo for "STANDALONE" in all the runtime scripts.
- Fixed incorrect environment variables used in ROUTER mode script.

**Comment by MatiasElo** [comment]
`pktio_env` is still missing from `example/ipsec_crypto/.gitignore`, but otherwise the PR looks good. You can add my reviewed-by tag while you fix that.

**Comment by asasidharan** [comment]
> 
> 
> `pktio_env` is still missing from `example/ipsec_crypto/.gitignore`, but otherwise the PR looks good. You can add my reviewed-by tag while you fix that.

Sorry, missed to update .gitignore file. Updated this with v3. Also added your Reviewed-by tag.
Thank you @MatiasElo for reviewing https://github.com/OpenDataPlane/odp/pull/1265 as well as this PR.

**Review by MatiasElo — APPROVED** [review]

**Pull request merged**